### DISCUSSION TITLE
Strimzi kafka migration between cluster k8s

### DISCUSSION THREAD

**Discussion by northboys** [strimzi/strimzi-kafka-operator#6793] [Q&A] [answered]
Hi, How to migration kafka from k8s cluster A to B?, there is example?

**Comment by scholzj** [comment, marked as answer]
I don't think there is any example as the best way will differ from environment to environment. You can use Mirror Maker, you can move the disks and start the new cluster using the old disks from the original cluster, you can use the `cold-backup` script from the `tools` directory. But which one is the best depends on your infrastructure, on how quickly you want to do it, what is the outage you are able to tolerate etc.

**Comment by veerendra2** [comment]
@northboys If the Kafka cluster is running on K8s that is powered by any cloud provider. You can use https://velero.io/ as they say

>Data Migration
>
>Enables cluster portability by easily migrating Kubernetes resources from one cluster to another​.
